### ISSUE TITLE
[thepiratebay] Exception (thepiratebay): An error occurred while sending the request: An error occurred while sending the request (Test)

### ISSUE THREAD

**Issue by glacier08** [Jackett/Jackett#1898]
Log file attached.
[log.txt](https://github.com/Jackett/Jackett/files/1338897/log.txt)

Mono version: 
Mono JIT compiler version 5.4.0.167 (tarball Wed 27 Sep 17:56:55 UTC 2017)

**Comment by kaso17** [comment]
This is often caused by missing CA certificates.
  Try reimporting the certificates in this case:
   - On Linux (as user root): `wget -O - https://curl.haxx.se/ca/cacert.pem | cert-sync /dev/stdin`
   - On macOS: `curl -sS https://curl.haxx.se/ca/cacert.pem | cert-sync --user /dev/stdin`

**Comment by SBado** [comment]
Same problem here (openSUSE Tumbleweed), even after reimporting the certificates.
[log.txt](https://github.com/Jackett/Jackett/files/1363037/log.txt)

Mono version:
Mono JIT compiler version 5.2.0.215 (tarball Tue Aug 29 11:47:00 UTC 2017)

**Comment by heyitsjames** [comment]
Sme problem here (MacOS 10.11.6), also after reimporting the certs. 

Mono version:
`Mono JIT compiler version 5.2.0.224 (d15-3/14f2c81 Thu Aug 24 10:33:52 EDT 2017)`

This happens when trying to add `1337x` for me.

**Comment by kaso17** [comment]
@SBado @heyitsjames are other HTTPS trackers working for you (1337x/thepiratebay)?

**Comment by SBado** [comment]
Yes, for example KickAssTorrent (https://katcr.co/) is working fine for me

**Comment by kaso17** [comment]
Which Site Link did you configure?

Please provide the output of the following two commands:
`csharp -e 'Console.WriteLine (new System.Net.WebClient ().DownloadString ("https://thepiratebay.org/"))'`
`openssl s_client -connect thepiratebay.org:443`

**Comment by SBado** [comment]
I think I found the problem: my ISP DNS. I didn't know I hosted thepiratebay.org...

`ping thepiratebay.org
PING thepiratebay.org (127.0.0.1) 56(84) bytes of data.
64 bytes from localhost (127.0.0.1): icmp_seq=1 ttl=64 time=0.065 ms
64 bytes from localhost (127.0.0.1): icmp_seq=2 ttl=64 time=0.063 ms
64 bytes from localhost (127.0.0.1): icmp_seq=3 ttl=64 time=0.068 ms`

After changing the DNS everything is working fine, thanks a lot and sorry if I wasted your time.
